frontend/javascripts/oxalis/view/action-bar/toolbar_view.tsx

```diff
@@ -233,26 +235,62 @@ function OverwriteModeSwitch({
   );
 }
 
+const INTERPOLATION_ICON = {
+  [InterpolationModeEnum.INTERPOLATE]: <i className="fas fa-align-center fa-rotate-90" />,
+  [InterpolationModeEnum.EXTRUDE]: <i className="fas fa-align-justify fa-rotate-90" />,
+};
 function VolumeInterpolationButton() {
   const dispatch = useDispatch();
+  const interpolationMode = useSelector(
+    (state: OxalisState) => state.userConfiguration.interpolationMode,
+  );
 
-  const onClick = () => {
+  const onInterpolateClick = (e: React.MouseEvent<HTMLButtonElement> | null) => {
+    e?.currentTarget.blur();
     dispatch(interpolateSegmentationLayerAction());
   };
 
   const { tooltipTitle, isDisabled } = useSelector((state: OxalisState) =>
     getInterpolationInfo(state, "Not available since"),
   );
 
-  return (
-    <ButtonComponent
-      disabled={isDisabled}
-      title={tooltipTitle}
-      icon={<i className="fas fa-align-center fa-rotate-90" style={{ marginLeft: 4 }} />}
-      onClick={onClick}
-      style={{ marginLeft: 12 }}
+  const menu = (
+    <Menu
+      onClick={(e) => {
+        dispatch(updateUserSettingAction("interpolationMode", e.key as InterpolationMode));
+        onInterpolateClick(null);
+      }}
+      items={[
+        {
+          label: "Interpolate current segment",
+          key: InterpolationModeEnum.INTERPOLATE,
+          icon: <i className="fas fa-align-center fa-rotate-90" />,
```

Comment:
You could also use the `INTERPOLATION_ICON` object here and in line 272 to avoid code duplication